src/robot/dial.ts

```diff
@@ -156,6 +156,7 @@ export const createRobotClient = async (
   conf: DialConf
 ): Promise<RobotClient> => {
   let client;
+  conf.authEntity = conf.authEntity?.replace(/^(?<http>.*:\/\/)/u, '');
```

Comment:
let's use the built-in URL parser to get the host

```suggestion
  if (conf.authEntity) {
    conf.authEntity = new URL(conf.authEntity).host;
  }
```